vms/platformvm/execconfig/execution_config.go

```diff
@@ -0,0 +1,51 @@
+package execconfig
```

Comment:
nit: platform vm already has a config package. I wonder if these files should be moved there, instead of creating a second, config-related package which may be confusing.
I'd  like to have a single config package with proper naming and comments for the two different configs which will live there.